app/javascript/admin/controllers/config_controller.js

```diff
@@ -93,6 +93,40 @@ export default class ConfigController extends Controller {
     }
   }
 
+  displaySnackbar(message) {
+    return document.dispatchEvent(
+      new CustomEvent('snackbar:add', { detail: { message } }),
+    );
+  }
+
+  async activateLightConfirmation(event) {
+    event.preventDefault();
+    try {
+      const data = new FormData(event.target);
+      const response = await fetch(event.target.action, {
+        method: 'POST',
+        headers: {
+          Accept: 'application/json',
+          'X-CSRF-Token': document.querySelector("meta[name='csrf-token']")
+            ?.content,
+        },
+        body: data,
+        credentials: 'same-origin',
+      });
```

Comment:
suggestion (not blocking) - you could use the `request` utility function from `app/javascript/utilities/http/request.js`, which adds a lot of these headers & the csrf token for you 🙂

Reply:
Ahhh ok, how do I do that? Is there an example in the codebase?

Reply:
Refactor (blocking): Plus one on the recommendation to use `request`. Using `request` encapsulates the required headers etc.

Reply:
Search the codebase for ` request(` (space before request), and you'll find a bunch of examples.